trigger_systemtests.py

```diff
@@ -39,20 +38,55 @@ def get_json_response(url, **kwargs):
 
     return json_response
 
+def adjust_travis_script(script, user, adapter):
```

Comment:
Maybe rename `adapter` to `repository` for generality?

Reply:
Actually `adapter` here is just the name of the adapter (e.g. 'calculix'). I then query real repository name from `nm_repo_map`